content/en/docs/reference/glossary/infrastructure-resource.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Resource (infrastructure)
+id: infrastructure-resource
+date: 2025-02-09
+short_description: >
+  A defined amount of infrastructure available for consumption (CPU, memory, etc).
+
+aka:
+tags:
+- fundamental
```

Comment:
(IMO)
```suggestion
- architecture
```

Reply:
(fixed the removed `tags:` in commit below)